signer/src/stacks/contracts.rs

```diff
@@ -311,12 +319,15 @@ impl CompleteDepositV1 {
     /// 5. That the recipients in the transaction matches that of the
     ///    deposit request.
     /// 6. That the amount to mint does not exceed the deposit amount.
-    /// 7. That the max-fee is less than the desired max-fee.
-    async fn validate_deposit_vars<C>(&self, ctx: &C, req_ctx: &ReqContext) -> Result<(), Error>
+    /// 7. That the fee matches the expected assessed fee for the output.
+    /// 8. That the max-fee is less than the desired max-fee.
```

Comment:
Those two points numbers are swapped wrt the docs of `CompleteDepositV1::validate`

Reply:
I fixed it in the `CompleteDepositV1::validate` docs in https://github.com/stacks-network/sbtc/pull/582/commits/985b75c78726bf9303d74a33877a75d80ea154fc